.github/actions/docker-setup/action.yaml

```diff
@@ -12,6 +12,10 @@ inputs:
   GCP_SERVICE_ACCOUNT_EMAIL:
     required: true
     description: "GCP service account email"
+  EXPORT_GCP_CREDENTIALS:
+    required: false
+    description: "Whether to export GCP credentials to the environment. Useful for running gcloud commands"
```

Comment:
btw, that name is misleading. This will not export credentials to the environment but rather just the default project variables.

Reply:
Renamed it to EXPORT_GCP_PROJECT_VARIABLES.